website/src/components/BlogListItem.tsx

```diff
@@ -21,6 +21,8 @@ export const BlogListItem: FunctionComponent<Props> = ({ blog }) => (
             <p className="text-muted">{blog.type}</p>
             <p>{blog.description}</p>
         </div>
-        <div className="col-lg-3">{blog.image && <img className="flex-1 blog-post__image" src={blog.image} />}</div>
+        <div className="col-lg-3">
+            {blog.image && <img className="flex-1 blog-post__image max-w-md-250" src={blog.image} />}
```

Comment:
I think we should be a bit more progressive with stacking elements and not stack so early, so that we can make use of as much screen real estate as possible for each breakpoint/device width. I know we aren't given tablet artboards in our Figma's, but we should start to think this way.

Thinking mobile-first, the images should stack and be full width below the small breakpoint, and then working your way upwards, do col-8/col-4 for the content/image respectively at the small breakpoint, and then 9/3 at the md breakpoint.

I think this component might need some rework. Try this:

For the root container:
```jsx
<div className="w-100 row mx-0 mb-5 pb-4 border-bottom">
```

For the content column:
```jsx
<div className="col-sm-8 col-md-9">
```

For the image column:
```jsx
<div className="col-sm-4 col-md-3 d-flex align-items-center">
            {blog.image && <img className="flex-1 w-100" src={blog.image} />}
```

At that point you can remove the styles for `&__image` in `_BlogPost.scss`

Reply:
I agree, thinking about tablet mid-size screens would be best practice. We may need to keep `&_image` though because it's applied on the `/blog` page. I just spent a bit trying to trade out some bootstrap classes for that page to get the same behavior but having a hard time since images have such varied sizes and dimensions. I can give it another look with a fresher brain next week.

Reply:
Oh good eye on the blog page! Ok, no worries! If it's too much of a hassle atm, we can tackle this during our Tailwind conversions. I know it's definitely a challenge to find and watch out for all the references with bem/global styles. 🙃